src/classes/Inventory.ts

```diff
@@ -193,7 +194,18 @@ export default class Inventory {
         return nonTradable.concat(tradable).slice(0);
     }
 
-    getAmount(sku: string, includeNonNormalized: boolean, tradableOnly?: boolean): number {
+    getAmount(
```

Comment:
With this function gaining parameters it maybe worth just making a params object that can handle optionally so that we don't have to make calls like (true, null, null, false)

Doesn't have to be but it occurred to me looking at all the touch points

Reply:
I've actually done that, but then when I see 90% need that, then I just make it not optional, but I will consider it.

Reply:
The refactor method in webstorm may make the work much more endurable however I'm unsure.

Check this out: https://www.jetbrains.com/help/webstorm/specific-javascript-refactorings.html#ws_js_destructuring_params_for_function

Reply:
Again, not saying it's necessary but if tempted..

Reply:
Thank you for the suggestion. The latest commits should resolve this :)